app/views/layouts/application.html.erb

```diff
@@ -7,20 +7,41 @@
     <%= csrf_meta_tags %>
   </head>
   <body>
+    <nav class="navbar navbar-default">
+      <div class="container-fluid">
+        <!-- Brand and toggle get grouped for better mobile display -->
+        <div class="navbar-header">
+          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
+            <span class="sr-only">Toggle navigation</span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+          </button>
+          <a class="navbar-brand" href="<%= root_path %>">The Dirty Cookie</a>
+        </div>
+        <!-- Collect the nav links, forms, and other content for toggling -->
+        <div class="collapse navbar-collapse navbar-right" id="bs-example-navbar-collapse-1">
+          <ul class="nav navbar-nav">
+            <li><a href="<%= menu_path %>">Menu<span class="sr-only">(current)</span></a></li>
+            <li><a href="<%= account_path %>">Your Account<span class="sr-only">(current)</span></a></li>
+            <li><a href="<%= cart_path %>"><span class="glyphicon glyphicon-shopping-cart" aria-hidden="true"></span></a></li>
```

Comment:
I had it in the erb block but then it looked like a link instead of part of the nav. We can ask about this, but I couldn't get it to work.